models/project/board.go

```diff
@@ -229,14 +229,14 @@ func UpdateBoard(ctx context.Context, board *Board) error {
 
 // GetBoards fetches all boards related to a project
 // if no default board set, first board is a temporary "Uncategorized" board
-func GetBoards(ctx context.Context, projectID int64) (BoardList, error) {
+func (p *Project) GetBoards(ctx context.Context) (BoardList, error) {
```

Comment:
Later on, we can even think about making this a `LoadBoards`, but that's for the future.